lib/Wikia/src/Service/User/Attributes/AttributeKeyValueService.php

```diff
@@ -59,14 +71,18 @@ public function get( $userId ) {
 			$this->error( $e->getMessage(), [
 				'user' => $userId
 			] );
-
-			throw $e;
 		}
 
 		return $attributeArray;
 	}
 
-	public function delete( $userId, $attribute ) {
+	/**
+	 * @param int $userId
+	 * @param Attribute $attribute
+	 * @return bool|true
```

Comment:
I don't think you need a pipe for two things that are basically the same, and also, this can return `false`, so you may as well just have `bool` here.